units/2023/unit_2023_03/english/AnaColon_summary_Enriquez_2003.md

```diff
@@ -0,0 +1,18 @@
+In Cuba at the time of the study there was only 28 known species of marine fungi. 20 of the known fungi are from the northwest region and the other 8 were reported by Enríquez y González. 
+
+The objective of the study was to determine the taxonomy of the marine fungi in six littoral beaches north of Havana. This would be added to the marine fungi collection at the Instituto de Oceanología for future study (Institue of Oceanography). 
+
+They classified the marine fungi into taxonomic level of the species using spore morphology. 
+
+They determined the frequency of the appearance of the species in terms of the number of individual isolates of one species compared to other species. 
+
+They identified 158 isolates and obtained 20 species of marine fungi from 180 samples. Of the 20 species 17 were of the phylum Ascomycota and 3 were "Mitosporic fungi". 
```

Comment:
apparently 'mitosporic fungi' are a catch-all group of species whose taxonomic position hasn't been established (or hadn't at the time of this paper). I suppose it shouldn't be capitalized because it is not a formal taxonomic group, so you could change that.